### PR TITLE
PaSh installation for Windows WSL

### PR DESCRIPTION
**Pull request by dkarnikis**
Step-by-step guide to setup WSL and install PaSh

### CODE REVIEW

File: docs/windows.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- Open PowerShell as Administrator and run:
+
+    `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`
+
+- Enable Linux Filesystem for Windows
+
+    `dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
+
+- Install the distro of your choice:
```
Comment (nvasilakis) [inline docs/windows.md:20]: Maybe only mention distros we have tried/supported by PaSh for now?

**Review by nvasilakis — APPROVED** [review]
Awesome work, thanks so much!

Maybe add a pointer from the main setup READMEs (top-level pash, and in /doc) to this file so that users find it? Maybe at the point where we have all distros?

File: docs/windows.md (hunk 1 above)
Comment (dkarnikis) [reply docs/windows.md:20]: Fixed!

File: docs/windows.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Windows Installation
+To setup and install PaSh on a Windows machine, `Windows Subsystem for Linux (WSL)` needs to be installed.
+
+## What is WSL
+The Windows Subsystem for Linux lets developers run a GNU/Linux environment 
+-- including most command-line tools, utilities, and applications -- 
+directly on Windows, unmodified, without the overhead of a traditional virtual 
+machine or dualboot setup.
+
+## Setup WSL
```
Comment (angelhof) [inline docs/windows.md:10]: I am not sure we should include such information in our documentation. The reason is that these instructions might quickly get outdated. Can we instead point to an official WSL installation link, e.g., https://docs.microsoft.com/en-us/windows/wsl/install-win10

**Pull request merged**